### ISSUE TITLE
Error 500 when request body is not what expected

### ISSUE THREAD

**Issue by alexzerah** [TimPrd/Be-Happik#20]
@TimPrd 
Concernant la route `/user/register`, 
si dans le body, le champ email n'est pas un tableau, alors le serveur envoie une erreur 500 `rawEmails.filter is not a function`

**Comment by TimPrd** [comment]
On peut checker et renvoyer plus propre mais il faut que ce soit un array d'email 😊 c'est la fonction du manager qui ajoute des gens.

**Comment by alexzerah** [comment]
Oui, je pensais à un check en back afin de renvoyer le message d'erreur. Sinon erreur 500 c'est pas top 🤕
